lib/fastlane/plugin/wpmreleasetoolkit/actions/android/an_validate_lib_strings_action.rb

```diff
@@ -41,17 +41,17 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :app_strings_path,
                                        description: 'The path of the main strings file',
                                        optional: false,
-                                       is_string: true),
+                                       type: String),
           FastlaneCore::ConfigItem.new(key: :libs_strings_path,
                                        env_name: 'CHECK_LIBS_STRINGS_PATH',
                                        description: 'The list of libs to merge',
```

Comment:
Would have been nice of us back then to be a bit more specific and better document what's expected here, as from what I see in the source code, this is not expected to just be an "array of strings" like one might think, but an "Array of Hashes", each hash in the array being expected to have certain specific keys.

Those keys are actually [documented in some YARD doc of the private method in the helper here](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_localize_helper.rb#L112-L118) but they are not documented in the action's own doc so that makes them hard to find and not being printed as part of the documentation when one would run `fastlane action an_validate_lib_strings`.

Up to you if you want to use the occasion of this PR to also update the description of this `ConfigItem` while at it, or do it in a separate PR, but in any case it would still be nice to fix at some point 🙂

Reply:
Added documentation here: https://github.com/wordpress-mobile/release-toolkit/pull/469/commits/1b6391c00d62bb2be8794583ff47d6cd37885fc7